src/codegen/ast.lisp

```diff
@@ -207,11 +211,15 @@ coalton symbols (`parser:identifier`)"
 
 (defstruct (node-application (:include node))
   "Function application (f x)"
-  (rator (util:required 'rator) :type node      :read-only t)
-  (rands (util:required 'rands) :type node-list :read-only t))
+  (inlinep   (util:required 'inlinep) :type boolean   :read-only t)
```

Comment:
Can we have one slot with some properties? Maybe a plist.

Then `node-application-inlinep` can be something like `(getf (node-application-properties node) ':inline)`.